.ci/appveyor.yml

```diff
@@ -63,7 +71,8 @@ test_script:
     "%CMD_IN_ENV% python -m pytest
     --cov -k "not ClangASTPrintBear and not ClangCloneDetectionBear and
     not ClangComplexityBear and not ClangCountVectorCreator and
-    not ClangCountingConditions"
+    not ClangCountingConditions and not RuboCopBearTest and
```

Comment:
yeah. it installs the gems but the tests fail for some reason in AppVeyor (link available above). same case for CSVLintBearTest